CPT331.Data.Migration/CPT331.Data.Migration.csproj

```diff
@@ -70,6 +70,12 @@
     <None Include="App.config">
       <SubType>Designer</SubType>
     </None>
+    <Content Include="CmdHelpers\dbprod.bat">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="CmdHelpers\dbdev.bat">
```

Comment:
Took me a while to workout what was required as cmd arguments. 
I thought a couple one-liner batch files would help out 😃